samples/boards/reel_board/mesh_badge/src/reel_board.c

```diff
@@ -517,24 +517,32 @@ static int configure_button(void)
 	}
 
 	gpio_pin_configure(gpio, DT_ALIAS_SW0_GPIOS_PIN,
-			   (GPIO_DIR_IN | GPIO_INT |  PULL_UP | EDGE));
+			   GPIO_INPUT | DT_ALIAS_SW0_GPIOS_FLAGS);
 
-	gpio_init_callback(&button_cb, button_interrupt, BIT(DT_ALIAS_SW0_GPIOS_PIN));
-	gpio_add_callback(gpio, &button_cb);
+	gpio_pin_interrupt_configure(gpio, DT_ALIAS_SW0_GPIOS_PIN,
+				     GPIO_INT_EDGE_TO_ACTIVE);
```

Comment:
It should be `GPIO_INT_EDGE_BOTH`, see the [definition](https://github.com/zephyrproject-rtos/zephyr/pull/22137/files#diff-8c357d91181b5a24a095015232ff4f1aL50) of EDGE, which is no longer needed BTW.

Reply:
Very well spotted. Fixed.